src/components/client/index/sections/ActiveCampaignsSection/ActiveCampaignCard/ActiveCampaignCard.tsx

```diff
@@ -63,19 +47,35 @@ export default function ActiveCampaignCard({ campaign, index }: Props) {
             : targetAmount.split('.')[0]}
         </Sum>
       </SumWrapper>
+      <Grid
+        sx={{
+          background: `url(${campaignImagesUrl})`,
```

Comment:
Not that it is going to be issue for now, but it is better to load images through Next's `<Image>` component when possible, as it offers some in-built image optimizations, which might result in slight performance improvement. This is just something to keep in mind. 

Leaving the necessary docs to read if you are interested.
[Next.js - Image Component](https://nextjs.org/docs/pages/api-reference/components/image)
[Next.js - Image Optimization](https://nextjs.org/docs/pages/building-your-application/optimizing/images)

Reply:
Great catch @sashko9807 ! We talked with @ani-kalpachka to make the image changes in separate PR to not delay the already ready visual improvements.

Reply:
Thanks for the review! 
I will apply this change in a separate PR.